test/FunctionalTests/Tests/DataOData/Tests/OData.Common.Tests/PublicApi/PublicApi.bsl

```diff
@@ -1932,6 +1932,11 @@ public sealed class Microsoft.OData.Edm.ExtensionMethods {
 	]
 	public static object[] GetAnnotationValues (Microsoft.OData.Edm.IEdmModel model, System.Collections.Generic.IEnumerable`1[[Microsoft.OData.Edm.Vocabularies.IEdmDirectValueAnnotationBinding]] annotations)
 
+	[
+	ExtensionAttribute(),
+	]
+	public static System.Collections.Generic.IList`1[[System.String]] GetDerivedTypeConstraints (Microsoft.OData.Edm.IEdmModel model, Microsoft.OData.Edm.Vocabularies.IEdmVocabularyAnnotatable target)

```

Comment:
>IList [](start = 42, length = 5)

This should be updated to IEnumerable<String> rather than IList<String> as per the updated code.